src/components/custom-select.tsx

```diff
@@ -48,9 +49,10 @@ export class CustomSelect extends React.PureComponent<IProps, IState> {
     const { items, HeaderIcon, isDisabled } = this.props;
     const currentItem = items.find(i => i === this.state.current);
     const showListClass = this.state.showList ? "show-list" : "";
+    const useHeader = this.props.isHeader ? "topHeader" : "";
```

Comment:
I would remove and make the default custom-select styling match what is needed in the header.